velox/functions/prestosql/aggregates/SimpleNumericAggregate.h

```diff
@@ -15,6 +15,7 @@
  */
 #pragma once
 
+#include "iostream"
```

Comment:
@karteekmurthys While we are at it, can you delete this line?

Reply:
Done.